network-autoconnections-httpks.ks.in

```diff
@@ -29,41 +37,54 @@ shutdown
 platform="$(get_platform @KSTEST_OS_NAME@ @KSTEST_OS_VERSION@)"
 # Note: up to RHEL8.2 the file was created by dracut network module
 # check_config_exists @KSTEST_NETDEV1@ yes
+# check_config_exists @KSTEST_NETDEV4@ yes
```

Comment:
Why not remove those commented-out lines and put the comment right before the condition below?